roles/common/tasks/monitoring.yml

```diff
@@ -225,3 +225,6 @@
     check: "{{ sensu_checks.common.check_raid }}"
   notify: restart sensu-client missing ok
   when: common.hwraid.enabled|bool
+
+- include: monitoring-vyatta.yml
+  hosts: controller
```

Comment:
That's now how you define which hosts this runs on. You'd need to put a conditional on this for 

`when: inventory_hostname in groups['controller']`

Also, there needs to be a conditional so that we only install these checks if the cloud has a vyatta (which some clouds do not).